velox/common/memory/Memory.h

```diff
@@ -86,7 +86,7 @@ struct MemoryManagerOptions {
   ///
   /// NOTE: the arbitrator will only be created if its kind is set explicitly.
   /// Otherwise MemoryArbitrator::create returns a nullptr.
-  std::string arbitratorKind{};
+  std::string arbitratorKind{"NOOP"};
```

Comment:
I remember we had some discussions around this since in the earlier version of PR #5965 I did make the kinds public constants.

@xiaoxmeng Do you want to comment here? Thanks.

Reply:
How about we create a noop arbitrator if arbitratorKind is empty?